scripts/config.py

```diff
@@ -80,6 +83,7 @@ def __init__(self):
         # Note that indexes must be created on db 0 in redis, this is not configurable.
 
         self.memory_backend = os.getenv("MEMORY_BACKEND", 'local')
+
```

Comment:
Rollback this unrelated whitespace change

Reply:
done